src/Build.OM.UnitTests/Construction/ProjectTargetElement_Tests.cs

```diff
@@ -339,6 +343,49 @@ public void SetReturns()
             Assert.True(project.HasUnsavedChanges);
         }
 
+        /// <summary>
+        /// Parse invalid property under target
+        /// </summary>
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void ReadInvalidPropertyUnderTarget(bool enableNewBehavior)
+        {
+            using (TestEnvironment env = TestEnvironment.Create())
+            {
+                ChangeWaves.ResetStateForTests();
+
+                if (enableNewBehavior)
+                {
+                    env.SetEnvironmentVariable("MSBUILDDISABLEFEATURESFROMVERSION", ChangeWaves.Wave17_6.ToString());
```

Comment:
`enableNewBehavior` is in fact _disabling_ the new behavior 😉

Reply:
Yes, I'm thinging to update it as !enableNewBehavior just now.